edx_solutions_api_integration/imports/views.py

```diff
@@ -51,6 +51,9 @@ def existing(self, request):
 
         if not errors:
             self._enroll_user(data, errors)
+            user = data.get('user_object')
```

Comment:
Add user id in the `_enroll_user` method same as `_create_user` is doing

Reply:
Addressed

Reply:
@naeem91 this needs to be reviewed too.